roofit/roofitcore/inc/RooDataHist.h

```diff
@@ -82,7 +76,6 @@ class RooDataHist : public RooAbsData, public RooDirItem {
   const RooArgSet* get() const override { return &_vars; }
   const RooArgSet* get(Int_t binNumber) const override;
   virtual const RooArgSet* get(const RooArgSet& coord) const;
-  Int_t numEntries() const override;
```

Comment:
`numEntries` was removed because it called `numEntries` of the base class anyway